vendors/maddalena/models/arrowwan2/model.yaml

```diff
@@ -0,0 +1,46 @@
+# Commercial name of the model 
+name: ArrowWan 2
+# Functional description of the product. Maximum 500 characters. 
+description: "Compact wMBus and LoRaWAN™ radio module, ensuring a high level of interoperability with a range of mobile and fixed network reading systems, including third-party ones. Arrow WAN 2 is also suitable for difficult Utility installations (IP68). Activation and configuration of the multiprotocol radio are carried out locally using the NFC interface and a dedicated Android app."
+# Logo of the device
+logo: arrowwan2.png
+# Product name & Version
+product:
+    name: MADDALENA/ARROWWAN2
+    version: 1
+# ID(s) of the profile that defines the LoRaWAN characteristics of this model.
+# id: as defined under /profiles.
+# lorawanDeviceProfileID: as defined by LoRa Alliance in https://lora-alliance.org/wp-content/uploads/2020/10/LoRa_Alliance_Vendor_ID_for_QR_Code_02142022.pdf.
+# The lorawanDeviceProfileID consists of 8 upper-case hexadecimal characters, 4 characters for VendorID + 4 characters for VendorProfileID. Please consider using single (') or double (") quotation.
+deviceProfileIds:
+ 
```

Comment:
Are there some device profiles missing? Strange to add only model without profiles. Is it requested to be used only with a custom device profile?